src/mainwindow.cpp

```diff
@@ -8948,9 +8948,13 @@ void MainWindow::handleScriptingNotesTagUpdating() {
 
     // workaround when signal blocking doesn't work correctly
     directoryWatcherWorkaround(true, true);
-
-    const QVector<Note> &notes = Note::fetchAll();
+    const QString defaultNoteFileExtension = Note::defaultNoteFileExtension();
+    const QVector<Note> notes = Note::fetchAll();
     for (const Note &note : notes) {
+        // ignore non-note files
+        if (note.fileNameSuffix() != defaultNoteFileExtension) {
```

Comment:
You can have an unlimited amount of note file extensions, not just the `defaultNoteFileExtension`!

Reply:
`Note::customNoteFileExtensionList` has that list.

Reply:
done.

Reply:
I will always wonder what is the use of such a feature